mixins/test/localize-mixin.test.js

```diff
@@ -142,15 +142,6 @@ describe('LocalizeMixin', () => {
 
 	});
 
-	describe('lang set', () => {
-
-		it('should ignore "__language" attribute and use default', async() => {
-			const elem = await fixture(html`<d2l-test-localize __language="fr"></d2l-test-localize>`);
-			expect(elem.__language).to.equal('en');
-		});
-
-	});
```

Comment:
This test was a holdover from the old `localize-behavior` days when the Polymer one used to sprout `__language` properties. Pointless to test for it now.